ManPac/Assets/Scripts/Player/Death/DeathHandler.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class DeathHandler : MonoBehaviour
+{
+
+    [SerializeField] private Animator GhostAnimator;
```

Comment:
maybe we should make it a convention to make the [SerializeField] above the variable
```suggestion
    [SerializeField]
    private Animator GhostAnimator;
```